templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Manufacturer</h1>
+  <p>Are you sure you want to delete the manufacturer {{ manufacturer }}?</p>
+  <p><i>All cars with this manufacture will be also deleted!</i></p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+  </form>

```

Comment:
Can you add a "No" or "Cancel" button? This is not a mistake, but it looks nicer, more complete